lib/SIL/Verifier/SILOwnershipVerifier.cpp

```diff
@@ -215,6 +233,79 @@ bool SILValueOwnershipChecker::isCompatibleDefUse(
   return false;
 }
 
+// Look for reborrows and find their base value and populate ReborrowVerifier's
+// worklist.
+void SILValueOwnershipChecker::populateReborrowVerifier(
```

Comment:
I would stick this in ReborrowVerifier. SILValueOwnershipChecker doesn't need to know about this detail.